public/app/features/variables/guard.ts

```diff
@@ -7,3 +7,8 @@ export const isQuery = (model: VariableModel): model is QueryVariableModel => {
 export const isAdHoc = (model: VariableModel): model is AdHocVariableModel => {
   return model.type === 'adhoc';
 };
+
+export const isMulti = (model: VariableModel): model is VariableWithMultiSupport => {
+  const withMulti = model as VariableWithMultiSupport;
+  return withMulti && typeof withMulti.multi === 'boolean';
```

Comment:
Maybe this would be a better approach, if model is undefined for some reason I would expect this to throw because it should never be undefined, right?
```suggestion
  return withMulti.hasOwnProperty('multi') && typeof withMulti.multi === 'boolean';
```